WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -295,4 +288,26 @@ public static void loginWithoutMagicLink(Activity activity) {
         signInIntent.putExtra(SignInActivity.EXTRA_INHIBIT_MAGIC_LOGIN, true);
         activity.startActivityForResult(signInIntent, RequestCodes.DO_LOGIN);
     }
+
+    /*
+     * open the passed url in the device's external browser
+     */
+    public static void openUrlExternal(Context context, @NonNull String url) {
+        try {
+            // disable deeplinking activity so to not catch WP URLs
+            WPActivityUtils.disableComponent(context, ReaderPostPagerActivity.class);
+
+            Uri uri = Uri.parse(url);
+            Intent intent = new Intent(Intent.ACTION_VIEW, uri);
+            context.startActivity(intent);
+            AppLockManager.getInstance().setExtendedTimeout();
+
+        } catch (ActivityNotFoundException e) {
+            String toastErrorUrlIntent = context.getString(R.string.no_default_app_available_to_load_uri);
+            ToastUtils.showToast(context, String.format(toastErrorUrlIntent, url), ToastUtils.Duration.LONG);
```

Comment:
I think we can drop the URL from the toast here. It makes it appear as though the URL itself might be the problem, which it isn't, and regardless the user is unlikely to have time to read and remember what the URL is.

![screenshot_1481583053](https://cloud.githubusercontent.com/assets/3903757/21120289/8df3d1ce-c094-11e6-8da3-2b9308a142f4.png)